documentation/src/main/asciidoc/user_guide/cdi.adoc

```diff
@@ -263,50 +290,68 @@ public class Config {
 With the above code the GreetingCache or the RemoteGreetingCache will be associated with the produced cache manager. 
 
 .Producer method scope
-NOTE: To work properly the producers must have the scope @ApplicationScoped . Otherwise each injection of cache will be associated to a new instance of cache manager. 
+NOTE: To work properly the producers must have the scope @ApplicationScoped . Otherwise each injection of cache will be associated to a new instance of cache manager.
 
 === Use JCache caching annotations
 
 TIP: There is now a separate module for JSR 107 (JCACHE) integration, including API.  See <<_using_infinispan_as_a_jsr107_jcache_provider, this chapter>> for details.
 
-The infinispan-cdi module provides a partial support of JCache caching annotations.
+The `infinispan-jcache` and `infinispan-jcache-remote` modules provide partial support of JCache caching annotations with CDI.
```

Comment:
There need to be both artifacts (JCache and CDI) on the classpath.. let me rephrase it..